recipes/libzip/all/conanfile.py

```diff
@@ -114,8 +114,13 @@ def _patch_sources(self):
         top_cmakelists = os.path.join(self.source_folder, "CMakeLists.txt")
         # Honor zstd enabled
         if self._has_zstd_support:
+            def zstd_find_package_pattern(version):
+                if version >= "1.9.2":
```

Comment:
@uilianries, but I pass Version(self.version) into this function already :-) How would it work otherwise? Do you really want to do this inside instead of outside?